### PR TITLE
intro: add sumator class

### PR DESCRIPTION
**Pull request by davidkostov**
Added a new sumator class that summs two integer numbers.

### CODE REVIEW

File: src/com/clouway/into/Sumator.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.clouway.into;
+
+/**
+ * Created by clouway on 06.07.17.
+ */
+public class Sumator {
+
+    public int sum(int a, int b){
```
Comment (mgenov) [inline src/com/clouway/into/Sumator.java]: Please add docs for this method.

File: src/com/clouway/into/Sumator.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.clouway.into;
+
+/**
+ * Created by clouway on 06.07.17.
```
Comment (mgenov) [inline src/com/clouway/into/Sumator.java]: You need to use standard java doc style for naming of the author. 

http://www.oracle.com/technetwork/articles/java/index-137868.html
In  section: Example Doc Comment 

Internally we are using something like:

```java
/**
 * @author Miroslav Genov (miroslav.genov@clouway.com)
 */
```

Comment (davidkostov) [inline src/com/clouway/into/Sumator.java]: Fixed.

Comment (davidkostov) [inline src/com/clouway/into/Sumator.java]: Fixed.

**Pull request merged**